File: SampleCode-V5/android-sdk-v5-uxsdk/src/main/java/dji/v5/ux/payload/data/DataSendMode.java
package dji.v5.ux.payload.data;

/**
 * 数据发送模式枚举
 */
public enum DataSendMode {
    MANUAL_ONLY,      // 仅发送手动输入的数据
    AUTO_ONLY,        // 仅自动发送PSDK数据
    BOTH              // 同时发送手动数据和PSDK数据
}

File: SampleCode-V5/android-sdk-v5-sample/src/main/java/dji/sampleV5/aircraft/data/DataSendMode.kt
package dji.sampleV5.aircraft.data

/**
 * 数据发送模式枚举
 * 
 * @author Payload开发
 * @date 2024/1/1
 * 
 * Copyright (c) 2024, DJI All Rights Reserved.
 */
enum class DataSendMode {
    MANUAL_ONLY,      // 仅发送手动输入的数据
    AUTO_ONLY,        // 仅自动发送PSDK数据
    BOTH              // 同时发送手动数据和PSDK数据
}